README.md

```diff
@@ -175,6 +175,8 @@ Configuration format is described in detail in the
 * `cartridge_enable_tarantool_repo` (`boolean`, optional, default: `true`):
   indicates if the Tarantool repository should be enabled (for packages with
   open-source Tarantool dependency);
+* `cartridge_remove_temporary_files` (`boolean`, optional, default: `false`):
+  indicates if temporary files should be removed;
```

Comment:
Maybe we should notice that user can add his own temporary files to `tmpfiles_list`?